infrastructure/lambda/template.yaml

```diff
@@ -0,0 +1,1384 @@
+    Description: >
+      The ARN of the Code Signing Config to use, provided by the deployment pipeline
+  PermissionsBoundary:
+    Type: String
+    Description: >
+      The ARN of the permissions boundary to apply to any role created by the template
+    Default: "none"
+  Environment:
+    Description: "The environment type"
+    Type: "String"
+    AllowedValues:
+      - "dev"
+      - "build"
+      - "staging"
+      - "integration"
+      - "production"
+    ConstraintDescription: must be dev, build, staging, integration or production
+  # When common stack support is added CriIdentifier should be pointed to passport SSM Param /common-cri-parameters/CriIdentifier
+  CriIdentifier:
+    Type: String
+    Default: "di-ipv-cri-uk-passport-api"
+
+Conditions:
+  EnforceCodeSigning: !Equals
+    - !Ref  CodeSigningEnabled
+    - true
+  CreateDevResources: !Equals
+    - !Ref Environment
+    - dev
+  IsProdEnvironment: !Equals
+    - !Ref Environment
+    - production
+  IsStubEnvironment: !Or
+    - !Equals [ !Ref Environment, dev]
+    - !Equals [ !Ref Environment, build ]
+    - !Equals [ !Ref Environment, staging ]
+    - !Equals [ !Ref Environment, integration ]
+  IsProdLikeEnvironment: !Or
+    - !Equals [ !Ref Environment, staging ]
+    - !Equals [ !Ref Environment, integration ]
+    - !Equals [ !Ref Environment, production ]
+  IsDevEnvironment: !Equals
+    - !Ref Environment
+    - dev
+  IsNotDevEnvironment: !Not
+    - Condition: IsDevEnvironment
+
+  AddProvisionedConcurrency: !Not
+    - !Equals
+      - !FindInMap [ProvisionedConcurrency, Environment, !Ref 'Environment']
+      -  0
+
+Globals:
+  Function:
+    VpcConfig:
+      SecurityGroupIds:
+        - !ImportValue cri-vpc-LambdaSecurityGroup
+      SubnetIds: !Split [ ",", !ImportValue cri-vpc-PrivateSubnets ]
+    CodeSigningConfigArn: !If
+      - CreateDevResources
+      - !Ref AWS::NoValue
+      - !Ref CodeSigningConfigArn
+    Timeout: 40
+    MemorySize: !FindInMap [ MemorySizeMapping, Environment, !Ref 'Environment' ]
+    Architectures:
+      - arm64
+    Environment:
+      Variables:
+        JAVA_TOOL_OPTIONS: -XX:+TieredCompilation -XX:TieredStopAtLevel=1
+        AWS_STACK_NAME: !Sub ${AWS::StackName}
+        POWERTOOLS_LOG_LEVEL: INFO
+        POWERTOOLS_METRICS_NAMESPACE: !Ref CriIdentifier
+
+Mappings:
+
+  AuthCodeExpirySeconds:
+    Environment:
+      dev: 600
+      build: 600
+      staging: 600
+      integration: 600
+      production: 600
+
+  MaximumAttemptCount:
+    Environment:
+      dev: 2
+      build: 2
+      staging: 2
+      integration: 2
+      production: 2
+
+  ProvisionedConcurrency:
+    Environment:
+      dev: 0
+      build: 0
+      staging: 1
+      integration: 1
+      production: 1
+
+  MemorySizeMapping:
+    Environment:
+      dev: 512
+      build: 1024
+      staging: 1024
+      integration: 1024
+      production: 2048
+
+  SessionTtlMapping:
+    Environment:
+      dev: "172800" # 2 days
```

Comment:
How come this is 2 days rather than 2 hours?

Reply:
This part of the template is aligned pre-common-cri-framework to 2 days.

Originally in passport this was 1 day (named backendSessionTtl), and the other CRI's had it at the time set to 2 days as SessionTtl

```
  backendSessionTtl:
    Type: AWS::SSM::Parameter
    Properties:
      Name: !Sub /${Environment}/credentialIssuers/ukPassport/self/backendSessionTtl
      Type: String
      Value: 86400
```

FraudCRI removing the SessionTtlMapping from template when it was 2 days.
https://github.com/alphagov/di-ipv-cri-fraud-api/commit/cc99b257eb32b570a678743634043892fa52fcb3

Checking the common-cri template SessionTtl is now set to 2 hours, I was unaware of this reduction.

Reply:
The SessionTtlMapping values are reduced to 7200.
A small investigation into the TTL found that the backendSessionTtl did operate in the same way as SessionTtl in common.
A review of TTL's has taken place with the VC ttl values now the same as other CRI's and the passport specific AccessTokenReqJwtTtl exposed.